lib/sequel/plugins/nested_attributes.rb

```diff
@@ -171,6 +172,30 @@ def nested_attributes_create(meta, attributes)
           obj = reflection.associated_class.new
           nested_attributes_set_attributes(meta, obj, attributes)
           delay_validate_associated_object(reflection, obj)
+          nested_attributes_add_association_object(reflection, obj)
+        end
+
+        # Find object with the given attributes, and add it to associations
+        # Returns the object found.
+        def nested_attributes_find(meta, attributes)
+          reflection = meta[:reflection]
+          obj = reflection.associated_class.find(attributes)
+          # Additional compliance checking of attributes
+          return if !obj || attributes.any? { |k, v| obj.values[k] != v }
```

Comment:
It works wrong without `attributes.any?`, look for [this subject](https://groups.google.com/forum/#!topic/sequel-talk/R1dt7MSbKr0).